clients/go/grpc/grpc.go

```diff
@@ -17,7 +17,7 @@ import (
 // NewGrpc creates the new grpc client with provided options.
 func NewGrpc(options ...GrpcOption) (*GrpcClient, error) {
```

Comment:
@AkbaraliShaikh would it better to expose constructors from the main client package instead of namespacing? Or you see any pros of this?

Reply:
We can do that, but I'll have to move all of the code to a single file client. Otherwise, I'll receive the cyclic dependency error.

I considered separating into distinct namespaces.

Reply:
And also I wanted the same name for options like 'WithHeader()'. If I remove it from the namespace, I must rename it for each client.

Reply:
Option consistency makes sense. For now, we can go ahead with it but we can take feedback on this from users.